### DISCUSSION TITLE
NetworkObject (Prediction 2) - Reset Smoothing or Force Teleport Function

### DISCUSSION THREAD

**Discussion by AavaGames** [FirstGearGames/FishNet#632] [Feature Request]
When [teleporting ](https://fish-networking.gitbook.io/docs/manual/guides/prediction/version-1/more/teleporting) reconciled objects there is currently no way to skip the smoothing that occurs unless `Enable Teleport` is active and they teleport are beyond the Teleport Threshold. A function that skips the past smoothing and cleans the slate similar to ClearReplicateCache() would be beneficial.

---

In my use case as an example: My Player's Pawn teleports back to a spawn point on the map when they are killed. Using `Enable Teleport` means that if they died close to their spawn they would smooth back into position looking odd. Also enable teleport causes issues for me on the Owner when the server updates them after they are hit by skills and begin to fly in a direction, in certain circumstances this distance can be quite large causing a disorienting teleport to the reconciled position.

**Comment by FirstGearGames** [comment]
In 4.2.1 smoothing got a big update. If you are still having this problem please create an issue here https://github.com/FirstGearGames/FishNet/issues

I realize this isn't exactly a bug but seems like it's important enough to get priority.

**Comment by FirstGearGames** [comment]
No longer applicable.
